docs/public/specifications/tests/Nameserver-TP/nameserver15.md

```diff
@@ -0,0 +1,119 @@
+# NAMESERVER15: Checking for software version
+
+## Test case identifier
+
+**NAMESERVER15**
```

Comment:
I think there should be no empty line here. I might not matter, but we have scripts reading the files.